example/filters.py

```diff
@@ -0,0 +1,12 @@
+import django_filters
+
+from example.models import Comment
+
+
+class CommentFilter(django_filters.FilterSet):
```

Comment:
Can you provide a link that shows an example?

DRF filtering depends on `django_filter` and similar examples are found in DRF docs.